### ISSUE TITLE
Epub not showing up on kindle

### ISSUE THREAD

**Issue by GuilhermeGomes1929** [aerkalov/ebooklib#285]
I made an ebook with this library.
I passed it to Kindle by Usb, but the epub file does not appear on Kindle. 
Apparently the epub generated is not compatible with Kindle. Some one can help me?

**Comment by VSP-HungPV** [comment]
You should convert epub to mobi or azw using Calibre. It's better for kindle.

**Comment by aerkalov** [comment]
Yeah, Kindle works with MOBI format and not EPUB. The best way to translate file to Kindle (or any other reader) is using Calibre app. It will also convert your EPUB file into MOBI and transfer it to the device. It will take care of some quirks some readers have with formatting, can cleanup, edit metadata and etc.
